src/PythonClient/src/quixstreams/models/streamproducer/timeseriesbufferproducer.py

```diff
@@ -18,18 +18,20 @@
 @nativedecorator
 class TimeseriesBufferProducer(TimeseriesBuffer):
     """
-        Class used to write to StreamProducer in a buffered manner
+    A class for writing time-series data to a StreamProducer in a buffered manner.
```

Comment:
looks like there is some leftover writing.

Also, we decided timeseries to be a single word for our usage, that is why it is not TimeSeriesData, but TimeseriesData.
Means no hypening it either

Reply:
I changed time-series to timeseries everywhere we had in in code. Few places only